tasks/mock-github

```diff
@@ -40,6 +41,10 @@ class Handler(MockHandler):
             })
         elif self.path == f'/{repo}/{sha}/.cockpit-ci/container':
             self.replyData('ghcr.io/cockpit-project/tasks')
```

Comment:
Kept the old path because I think we can't easily land this green otherwise.

Reply:
I agree, but maybe you change the subject of the commit message to reflect that you're adding a new endpoint...